lib/scala/pkg/src/main/java/module-info.java

```diff
@@ -0,0 +1,12 @@
+module org.enso.pkg {
+  requires scala.library;
+  requires org.apache.commons.compress;
+  requires org.enso.editions;
+  requires org.enso.semver;
+  requires org.enso.scala.yaml;
```

Comment:
Wouldn't it be better to `requires transitive org.enso.scala.yaml` rather then adding the next two libraries here as well?